network/vpc/troubleshooting/autoconfig-not-working.mdx

```diff
@@ -1,27 +1,27 @@
 ---
 meta:
-  title: My Instance is attached to a Private Network but autoconfiguration is not working
-  description: This page helps you troubleshoot problems when autoconfiguration of your Instance on a Private Network is not working
+  title: My Instance is attached to a Private Network but auto-configuration is not working
+  description: This page helps you troubleshoot problems when auto-configuration of your Instance on a Private Network is not working
 content:
-  h1: My Instance is attached to a Private Network but autoconfiguration is not working
-  paragraph: This page helps you troubleshoot problems when autoconfiguration of your Instance on a Private Network is not working
-tags: resource instance private-network autoconfiguration auto configuration dhcp default-route
+  h1: My Instance is attached to a Private Network but auto-configuration is not working
+  paragraph: This page helps you troubleshoot problems when auto-configuration of your Instance on a Private Network is not working
+tags: resource instance private-network auto-configuration auto configuration dhcp default-route
 dates:
-  validation: 2023-07-05
+  validation: 2024-01-08
   posted: 2023-07-05
 categories:
   - network
 ---
 
-Default Instance images for Ubuntu, Debian and CentOS support autoconfiguration of interfaces plugged into a Private Network.
+Default Instance images for Ubuntu, Debian and CentOS support auto-configuration of interfaces plugged into a Private Network.
 
-They leverage helper scripts provided by the scaleway-ecosystem package. These scripts:
+They leverage helper scripts provided by the `scaleway-ecosystem` package. These scripts:
 
 - enable DHCP on the interfaces plugged into a Private Network
 - when Instances are attached to Private Networks plugged into Public Gateways, make the default route received by DHCP the primary route for all traffic on the Instance
 - keep the route to the Scaleway Metadata API more specific (see below).
 
-If your Instance does not get autoconfigured, it may be that you are using an old version of the scaleway-ecosystem package. `scaleway-ecosystem 0.0.4` or later is required. Use the following command to update it:
+If your Instance does not get auto-configured, it may be that you are using an old version of the `scaleway-ecosyste`m package. `scaleway-ecosystem` version 0.0.4 or later is required. Use the following command to update it:
```

Comment:
```suggestion
If your Instance does not get auto-configured, it may be that you are using an old version of the `scaleway-ecosystem` package. `scaleway-ecosystem` version 0.0.4 or later is required. Use the following command to update it:
```